llvm/test/CodeGen/PowerPC/vsx-fma-mutate-trivial-copy.ll

```diff
@@ -1,4 +1,4 @@
-; RUN: llc -verify-machineinstrs < %s | FileCheck %s
+; RUN: llc -verify-machineinstrs -fp-contract=fast < %s | FileCheck %s
```

Comment:
Can you drop the unsafe-fp-math attributes in these tests?

Reply:
PowerPC is special, `PPCTargetLowering` depends on this options to set correct lowering action (for some sqrt operations).